data_processors/pipeline/domain/tests/test_workflow.py

```diff
@@ -16,6 +16,16 @@ class WorkflowDomainUnitTests(PipelineUnitTestCase):
     def setUp(self) -> None:
         super(WorkflowDomainUnitTests, self).setUp()
 
+    def test_portal_run_id(self):
+        """
+        python manage.py test data_processors.pipeline.domain.tests.test_workflow.WorkflowDomainUnitTests.test_portal_run_id
+        """
+        helper = WorkflowHelper(WorkflowType.BCL_CONVERT)
+        logger.info(helper.get_portal_run_id())
+        self.assertIsNotNone(helper.portal_run_id)
+        self.assertEqual(len(helper.portal_run_id), 16)
+        self.assertEqual(helper.get_portal_run_id(), helper.portal_run_id)
```

Comment:
Could add a test that two IDs from two helpers are not the same?
I.e. test that the ID is actually created new each time

Reply:
Improved with https://github.com/umccr/data-portal-apis/commit/6232b6f259858a85e4a6838d6a7d3eb7189e37e9